tests/unit/h/security/policy/_cookie_test.py

```diff
@@ -93,6 +127,28 @@ def test_methods_add_vary_callback(
             == Any.iterable.containing(expected_vary).only()
         )
 
+    @pytest.fixture
+    def cookie(self, user):
+        cookie = create_autospec(
+            webob.cookies.SignedCookieProfile, instance=True, spec_set=True
+        )
+        cookie.get_value.return_value = (user.userid, sentinel.ticket_id)
+        return cookie
+
+    @pytest.fixture
+    def cookie_policy(self, cookie):
+        return CookiePolicy(cookie)
+
     @pytest.fixture
     def user(self, factories):
         return factories.User()
+
+
+@pytest.fixture(autouse=True)
+def urlsafe_b64encode(mocker):
+    return mocker.spy(base64, "urlsafe_b64encode")
+
+
+@pytest.fixture(autouse=True)
+def urandom(mocker):
+    return mocker.spy(_cookie, "urandom")
```

Comment:
Using `pytest-mock` spies here, so the tests actually call the real `urlsafe_b64encode()` and `urandom()` but they can also spy on these functions and make assertions about how they were called and what was done with their return values.